src/app/components/NavigationBar/NavigationBar.blocks.tsx

```diff
@@ -184,7 +185,6 @@ export const NavigationBarFull: React.FC<NavigationBarFullProperties> = ({
 	const scroll = useScroll();
 	const { openExternal } = useLink();
 	const { isLg, isMd } = useBreakpoint();
-	const availableNetworks = useNetworks({ profile });
 
```

Comment:
@dated I see that the list is not updated instantly after saving, maybe putting the following line here (above the hook) and adding it as a dep below might fix the reactivity..

```suggestion
const enabledNetworkIds = profileAllEnabledNetworkIds(profile);
```